docs/en_US/TrainingService/RemoteMachineMode.md

```diff
@@ -100,9 +100,10 @@ machineList:
   - ip: 10.1.1.3
     username: bob
     passwd: bob123
+    preCommand: source /home/bob/anaconda3/bin/activate nni-env
 ```
 
-Files in `codeDir` will be uploaded to remote machines automatically. You can run below command on Windows, Linux, or macOS to spawn trials on remote Linux machines:
+Files in `codeDir` will be uploaded to remote machines automatically. The `preCommand` will be executed before the remote machine executes other commands. So if you need to configure the experimental environment such as activate the conda environment, set `preCommand`. You can run below command on Windows, Linux, or macOS to spawn trials on remote Linux machines:
 
```

Comment:
suggest to use a subsection to explain how to config python environment with `preCommand`